r/README.md

```diff
@@ -0,0 +1,69 @@
+# R recommendations
+
+## Tidyverse style guide
+
+In general, we recommend following the [Tidyverse style guide](https://style.tidyverse.org/) as much
+as possible. There are two R packages that support this style guide out of the box:
+
+* [*styler*](http://styler.r-lib.org/) allows to interactively style selected text, files or entire projects.
+  It includes a handy RStudio plugin to make your R code beautiful with one simple keyboard shortcut
+* [*lintr*](https://github.com/jimhester/lintr) checks whether your code conforms to the style guide and flags
+  potential problems. It can also be enabled in RStudio.
+
+Both these tools can be customised to disable certain checks or using different values. However, we
+recommend sticking to the default values as much as possible.
+
+For VSCode users, the [vscode-R](https://code.visualstudio.com/docs/languages/r) plugin provides most
+of these features in VSCode as well.
+
+>A note on the use of the pipe (`|>` or `%>%`) operator, although
```

Comment:
```suggestion
> [!NOTE]
> A note on the use of the pipe (`|>` or `%>%`) operator, although
```
Will render a nice [callout, which GitHub calls an "alert" ni their flavour of markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts). 

But...

> [!WARNING]
>  I'm not certain it will play nicely with the jekyll site.